### ISSUE TITLE
Add support for pipeline branching based on manual judgement

### ISSUE THREAD

**Issue by bfarrell** [spinnaker/spinnaker#688]
Would like the ability to fork my pipeline with only one branch live.  Currently, trying to either rollback or delete the old instance depending on a manual judgement.
![image](https://cloud.githubusercontent.com/assets/3035661/12433810/cf661a78-bec8-11e5-8c7d-aef27ef0e018.png)
I tried adding Check Precondition stages in each parallel branch but because its triggered on a Manual Judgement, I can only check on judgmentStatus=continue to get one branch.  The other branch can't check for a stop.

Potential solution I thought of was to add an entry field to the manual judgement.  That way I could tag it as rollback or "rollforward" and do check precondition on that value.

I should mention I worked around this now with a manual judgement in each of the paths, but it opens it up for user error by someone selecting continue on both paths.

**Comment by cfieber** [comment]
I like this idea, and it doesn't seem like we already have most of the building blocks to implement it

**Comment by bfarrell** [comment]
I wrote this up at the end of last week and started playing with it.  I just created a free form text box.  It stores the value in the context, which I then evaluate in the Check Precondition.  It all works, but I have some concerns that it is error prone.  I was thinking about expanding it a bit and allowing users to add values in the design page which are then chosen through a drop down in the manual judgment.  Thoughts?
